src/Twig/GotenbergRuntime.php

```diff
@@ -14,6 +15,10 @@ final class GotenbergRuntime
 {
     private BuilderAssetInterface|null $builder = null;
 
+    public function __construct(private readonly Packages|null $packages)
```

Comment:
```suggestion
    public function __construct(private readonly Packages|null $packages =  null)
```
To  make it simpler to use (eg: avoid `new GotenbergRuntime(null)` in tests)

Reply:
Oups nice catch